Ports/SuperTuxKart/package.sh

```diff
@@ -39,8 +39,8 @@ configopts=(
     '-DUSE_DNS_C=ON'
     '-Bbuild'
     '-GNinja'
-    "-DCMAKE_C_FLAGS=-I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2 -I${SERENITY_INSTALL_ROOT}/usr/local/include -lcurl -lharfbuzz -lfreetype"
-    "-DCMAKE_CXX_FLAGS=-I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2 -I${SERENITY_INSTALL_ROOT}/usr/local/include -lcurl -lharfbuzz -lfreetype"
+    "-DCMAKE_C_FLAGS=-I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2 -lcurl -lharfbuzz -lfreetype"
```

Comment:
Diff looks fine once the other PR is in, but does this actually fix building? I would expect this to be a no-op after the other PR. Maybe "Remove now-unnecessary flag" is a better commit title?

Reply:
so the removed `-I` is `Additionally, removed legacy include paths.` from the commit message. The actual fix here is leaving `-lcurl -lharfbuzz -lfreetype` out of CXXFLAGS, because something along the chain does `-Werr` and clang treats unused libraries as errors in this case.

Should I reword the commit message anyways, or is this OK?

Reply:
Oh, I see. Fine as-is then, thanks for explaining.

Out of curiosity, is it clang saying `-l is unused for compilations`? In that case we could use `CMAKE_SHARED_LINKER_FLAGS` instead of `C{,XX}_FLAGS` – putting `-l` flags in CFLAGS is a bit weird. Or is it lld saying "-l flag passed but not needed"?

Reply:
That's the compiler saying that linker flags are unused when just compiling code. Using CMAKE_SHARED_LINKER_FLAGS (or, more likely, CMAKE_EXE_LINKER_FLAGS) for these flags should work too and seems like maybe a nicer fix.